README.md

```diff
@@ -147,12 +146,12 @@ config :mongoose_push, fcm: [
   ]
 ```
 
-Here we can see definition of a pool. Each pool has a name and its configuration. You may have several named pools of different sizes and with different configurations. Currently the only reason you may want to do this is that, the `REST` client may switch between them by specifying matching `:mode` in their push request.
+This is a definition of a pool - each pool has a name and configuration. It is possible to have multiple named pools with different configuration, which includes pool size, environment mode etc. Currently the only reason you may want to do this is to create separate production and development pools which may be selected by a `HTTP` client by specifying matching `:mode` in their push request.
 
 Each `FCM` pool may be configured by setting the following fields:
 * **key** (*required*) - you `FCM` Application Key for using Googles API
 * **pool_size** (*required*) - maximum number of used `HTTP/2` connections to google's service
-* **mode** (*either `:prod` or `:dev`*) - pool's mode. `REST` client may select pool used to push his notification by specifying matching option in his request
+* **mode** (*either `:prod` or `:dev`*) - pool's mode. The `HTTP` client may select pool used to push his notification by specifying matching option in the request
```

Comment:
The `HTTP` client may select a pool used to push notifications by specifying a matching option in the request